testresults/src/org/labkey/testresults/view/runDetail.jsp

```diff
@@ -143,13 +145,14 @@
     <% } %>
 </table>
 <% }
-if (testmemoryleaks.length > 0) { %>
+if (leaks.length > 0) { %>
 <table class="decoratedtable" style="float: left;">
-    <tr><td>Leaks</td><td>Bytes</td></tr>
-    <% for (TestMemoryLeakDetail l: testmemoryleaks) { %>
+    <tr><td>Leaks</td><td>Bytes</td><td>Handles</td></tr>
+    <% for (TestLeakDetail l: leaks) { %>
     <tr>
         <td><%=h(l.getTestName())%></td>
-        <td><%=h(l.getBytes()/1000 + "kb")%></td>
+        <td><% if (l instanceof TestMemoryLeakDetail) { %><%= h((Integer)(TestMemoryLeakDetail.class.getMethod("getBytes").invoke(l))/1000) %> kb<% } %></td>
```

Comment:
- Instead of using reflection can this be ((TestMemoryLeakDetail)l).getBytes()) ?
- No need to encode int